Orange/evaluation/testing.py

```diff
@@ -366,21 +365,21 @@ def __init__(self, data, learners, store_data=False, store_models=False,
         self.call_callback(1)
 
 
-class Bootstrap(Results):
```

Comment:
I wouldn't since this is not a change of the interface, but actually removal of Bootstrap and introduction of ShuffleSplit. They are similar enough that most of the code can be reused, but the method is different.

And since the class was used only in one place (and even there incorrectly), I don't think we need to bother with deprecating it first...